.env.example

```
@@ -45,6 +45,23 @@ STATUS_PAGE_PORT=3001
 # Domain where status pages are accessible via slug (e.g., status.uptimekit.com/acme)
 NEXT_PUBLIC_STATUS_PAGE_DOMAIN=status.uptimekit.com
 
+# ============================================
+# Subscriber Notifications
+# ============================================
+# Set to "resend" or "smtp" to enable subscriber email delivery
+EMAIL_PROVIDER=resend
+# Sender used for subscriber notification emails
+EMAIL_FROM=UptimeKit <status@example.com>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_FROM key should go before the EMAIL_PROVIDER key

(UnorderedKey)